ext/node/polyfills/01_require.js

```diff
@@ -241,8 +241,10 @@ setupBuiltinModules();
 const cjsParseCache = new SafeWeakMap();
 
 function pathDirname(filepath) {
-  if (filepath == null || filepath === "") {
+  if (filepath == null) {
     throw new Error("Empty filepath.");
+  } else if (filepath === "") {
+    return ".";
```

Comment:
Node's `path.dirname` function doesn't throw when it receives an empty string like we did. Instead it returns `"."`. See https://github.com/nodejs/node/blob/38e6ac7b446bcf2be257fde6763d1da7d2104295/lib/path.js#L660-L661

Reply:
Good catch! It appears this is correctly implemented in actual `path.dirname()` polyfill